charts/fybrik/values.yaml

```diff
@@ -50,6 +50,24 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # If deploying to OpenShift
```

Comment:
maybe "is the target deployment OpenShift"

Reply:
Fixed. Thanks